### PR TITLE
Create nikodonnell.json

### PR DESCRIPTION
**Pull request by nikodonnell**
I would like to register for JS CraftCamp.    
- [x] My Pull Request contains a json file `$firstname_$lastname.json` or `$nickname.json`    
- [ x] The json file follows the template at https://github.com/jscraftcamp/website/blob/master/participants/_template.json and contains the mandatory fields `name`, `tags`, `when`, `what_is_my_connection_to_javascript` and `what_can_i_contribute`    
- [ x] If I can't attend, I will either send another Pull Request removing my json file or an E-Mail with the subject 'UNREGISTER' to team@jscraftcamp.org

### CODE REVIEW

File: participants/nikolas_odonnell.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "when": {
+    "friday": true,
+    "friday_party": true,
+    "saturday": true
+  },
+  "email": "nikodonnell@gmail.com",
+  "dietary_requirements": "vegetarian",
+  "what_is_my_connection_to_javascript": "I love JS - I've chatted with Brendan Eich, when I was just starting out. I'm still a new developer, but working full time, most recently picking up React.",
+  "what_can_i_contribute": "I'm super enthusiatic, ask questions, and generally love web development and JavaScript so will be happy to learn as much as I can. Whilst working on side projects and listening to more seasoned professionals.",
+  "tshirt": "/^(W|M)-(M-L)$/",
```
Comment (davelosert) [inline participants/nikolas_odonnell.json]: Shirt Size needs to follow this regex. Possible values are for example `W-M` or `M-L`. 
The first letter is the shirt-gender-type (W=Women, M=Man), the second the size.

**Comment by davelosert** [comment]
@nikodonnell : Perfect now. Thanks for the fixes and see you at JSCC17!

**Comment by nikodonnell** [comment]
Hi @davelosert - I'm so sorry, I'm not able to make it, have to go to Vancouver for work and won't make it back in time. I was really looking forward, sorry! 

Let me know if I should do something else to confirm.

**Comment by davelosert** [comment]
@nikodonnell : That is too bad. Even though I am a little bit jealous: Vancouver is a beautiful city!
If you could just go ahead an open another Pull Request where you simply delete your registration-json that'd be great. If you need help or want me to do it just let me know.

In any case: Thank you very much for letting us know and have a lot of fun in vancouver!

If you register with our newsletter, you will be kept up to date with a possible JSCC18 next year - maybe it will work out then!

**Pull request merged**